mail_activity_team/models/mail_activity_team.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models, _
+
+
+class MailActivityTeam(models.Model):
+    _name = "mail.activity.team"
+    _description = 'Mail Activity Team'
+
+    name = fields.Char(string='Name', required=True, translate=True)
+    active = fields.Boolean(string='Active', default=True)
+    res_model_ids = fields.Many2many('ir.model', string='Used models')
```

Comment:
Could be convenient to use readonly on that field to avoid capacity to create models by managers randomly.